distro/ddl/pom.xml

```diff
@@ -0,0 +1,175 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>io.apiman</groupId>
+    <artifactId>apiman-distro</artifactId>
+    <version>1.1.4-SNAPSHOT</version>
+  </parent>
+  <artifactId>apiman-distro-ddl</artifactId>
+  <name>apiman-distro-ddl</name>
+
+  <properties>
+    <!-- Live Database Hosts-->
+    <hostname.postgresql.live.db>db-server.punagroup.com</hostname.postgresql.live.db>
+    <hostname.mysql.live.db>db-server.punagroup.com</hostname.mysql.live.db>
+    <!-- Database Driver Versions-->
+    <version.org.postgresql.jdbc.driver>9.4-1201-jdbc41</version.org.postgresql.jdbc.driver>
+    <version.mysql.jdbc.driver>5.1.35</version.mysql.jdbc.driver>
+  </properties>
+
+  <dependencies>
+    <!-- Project Dependencies -->
+    <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>apiman-manager-api-beans</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+
+    <!-- Third Party Dependencies -->
+    <dependency>
+      <groupId>org.hibernate.javax.persistence</groupId>
+      <artifactId>hibernate-jpa-2.1-api</artifactId>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.hibernate</groupId>
+      <artifactId>hibernate-entitymanager</artifactId>
+      <version>4.3.1.Final</version>
```

Comment:
I'll have experiment with this.  With regard to the apiman dependency, we don't want to inherit the whole apiman project classpath.  I ran into this issue before, the entity bean scan that we've configured for liquibase will discover other persistence.xml files and attempt to act on those and fail.  Goal here is to limit the classpath to include only entities.